apps/i18n/common/en_us.json

```diff
@@ -1095,7 +1095,7 @@
   "goToMyDashboard": "Go to my Dashboard",
   "goToUnit": "Go to Unit",
   "grade": "Grade",
-  "gradeRange": "Grades: {youngest_grade}-{oldest_grade}",
+  "gradeRange": "{numGrades, plural, one {Grade: {youngestGrade}} other {Grades: {youngestGrade}-{oldestGrade}}}",
```

Comment:
I don't expect you to change anything since this is probably a core issue with our platform, but this is just a thought I had: I imagine "Age" would be a more universal attribute across cultures rather than the specific US education concept of a "Grade".

Reply:
Yes, good call. We'll use the user's country to determine if we show `age` (outside the US) or `grade` (inside the US), but that's happening in version 2.